x-pack/plugins/observability_solution/slo/public/embeddable/slo/burn_rate/burn_rate.tsx

```diff
@@ -84,7 +74,7 @@ export function BurnRate({ sloId, sloInstanceId, duration, reloadSubject }: Embe
           <EuiFlexItem>
             <EuiLink
               data-test-subj="sloBurnRateLink"
-              className={link}
+              css={link}
```

Comment:
It would be nice to use EUI variables for the link CSS defined at the bottom of the page here instead of hard coded values. Something like:

```tsx
const link = ({ euiTheme }: UseEuiTheme) => css`
  font-size: ${euiTheme.size.base};
  font-weight: ${euiTheme.font.weight.bold};
`;